blazar/plugins/oshosts/host_plugin.py

```diff
@@ -199,6 +199,19 @@ def on_start(self, resource_id, lease=None):
                 parameters=dict(
                     reservation_id=host_reservation['reservation_id']))
 
+        for host in hosts:
+            for server in self.nova.servers.list(
+                    search_opts={"host": host, "all_tenants": 1}):
```

Comment:
This _for sure_ works / is safe? I know that we use `hypervisor_hostname` in a lot of places where nova normally uses `host`. Just want to make sure this isn't going to remove all user instances...

All Ironic hypervisors have host=`admin01-ironic` at UC for example, but all have different hypervisor hostnames. It _looks_ like this should be OK for us, just wondering if you tested this specifically (launch 1 node on preemptible, 1 node via lease, then reserve the preemptible node and ensure the node launched on the other lease doesn't have its instance terminated.)